File: docs/src/testers/scraping/simple.ts
// @sniptest filename=simple_scrape.ts
// @sniptest show=1-7
import { NotteClient } from 'notte-sdk';

const client = new NotteClient();
const markdown = await client.scrape('https://www.notte.cc', {
  only_main_content: true,
});
console.log(markdown);

export const results = [typeof markdown === 'string', markdown.toLowerCase().includes('notte')];

File: docs/src/testers/scraping/simple.py
# @sniptest filename=simple_scrape.py
# @sniptest show=1-8
from notte_sdk import NotteClient

client = NotteClient()
markdown = client.scrape(
    url="https://www.notte.cc",
    only_main_content=True,
)
print(markdown)

results = [isinstance(markdown, str), "notte" in markdown.lower()]
